opennms-config-tester/src/main/resources/META-INF/opennms/applicationContext-configTester.xml

```diff
@@ -238,7 +238,7 @@
     <property name="configResource" value="file:${opennms.home}/etc/jdbc-datacollection-config.xml"/>
   </bean>
 
-  <bean id="jmxConfigDao" class="org.opennms.netmgt.dao.jmx.JmxConfigDaoCm"/>
+  <bean id="jmxConfigDao" class="org.opennms.netmgt.dao.jaxb.DefaultJmxConfigDao"/>
 
```

Comment:
only one minor concern about the package name. It is not jaxb related, it is better to name it cm or something else.